### ISSUE TITLE
Question about graph embedding spaces

### ISSUE THREAD

**Issue by matentzn** [Knowledge-Graph-Hub/kg-phenio#53]
Hey all, working on our RPPR aims and I stumbled across the following question. Lets say we have this graph here:

<img width="577" alt="image" src="https://user-images.githubusercontent.com/7070631/174485201-90acb687-c7c2-4c66-a478-f9be06553f58.png">

I am assuming now that in KG-embedding space (word and graph embeddings), all three species-specific eyes have the same distance from each other - correct me if this assumption is wrong.

If I now were to layer on top this graph another graph:

<img width="828" alt="image" src="https://user-images.githubusercontent.com/7070631/174485427-4fb36fdc-f00e-4fdf-82a5-052f2037cddb.png">

Am I correct in assuming that this would result in Monkey and Human eyes being closer in embedding space than Human and Bird eyes? Thank you for you help :)

**Comment by LucaCappelletti94** [comment]
Yes, generally speaking that should hold true for most topological embedding methods. We can test this experimentally rather easily, I'll try it out later this afternoon.

**Comment by LucaCappelletti94** [comment]
Should I close this issue?

**Comment by LucaCappelletti94** [comment]
Ok so, so after having run a wide range of embeddings, I can say that there is no statistical difference.

More specifically:
- I have run SkipGram, CBOW, GloVe and TransE from Ensmallen
- I have run TransE and BoxE from PyKeen

In the small original graph, the nodes are all equally far apart. In the second graph, the cosine similarities change but not in a constant manner. In some runs and embeddings, the bird eye is closer than the human or monkey eye. I think it boils down to the fact that the three nodes have the same minimum path between each other, and that is what influences most the embedding, more than any other connection can.

**Comment by matentzn** [comment]
Ahhhh very unfortunate! :/ I was hoping to be able to trick graph embeddings into being able to recognise taxonomic distance.. THANK you for checking.

**Comment by caufieldjh** [comment]
What if we embed without the edges between "(species) eye" and "metazoan eye"?

**Comment by matentzn** [comment]
I think this would be a big hack.. The edges do matter when you have a massive ontology like uPheno.. Maybe they would decrease in importance if the example was somewhat larger..
